packages/design-picker/src/components/mshots-image/index.tsx

```diff
@@ -1,32 +1,42 @@
 /**
  * External dependencies
  */
-import React, { useState, useLayoutEffect } from 'react';
+import React, { useState, useEffect, useLayoutEffect } from 'react';
 import classnames from 'classnames';
 import { addQueryArgs } from '@wordpress/url';
 
 /**
  * Style dependencies
  */
+import { isEnabled } from '@automattic/calypso-config';
```

Comment:
Just a nit, this should have an "Internal dependencies" heading. You could just rename the "Style dependencies" one. (Personally I'm not really a fan of these import comments, but may as well keep them tidy :sigh:)

Reply:
https://github.com/Automattic/wp-calypso/pull/51261/commits/dfb1adaf7a2866181f7d7bbfdc6173402823821f